spring-cloud-zuul-ratelimit-tests/springdata/src/main/resources/bypass-request-location.properties

```diff
@@ -0,0 +1 @@
+zuul.ratelimit.location.bypass[0]=127.0.0.1
```

Comment:
Maybe we also need a test/configuration to validate that both the bypass and deny work at the same time.

Reply:
I rewrote the tests in a way to test it all together